### PR TITLE
Gaudi OpenShift notebook container

### PR DESCRIPTION
**Pull request by sharvil10**
This PR adds the Dockerfiles for notebook container for OpenShift AI.

### CODE REVIEW

File: enterprise/redhat/openshift-ai/gaudi/docker/docker-compose.yaml
Comment (tylertitsworth) [inline enterprise/redhat/openshift-ai/gaudi/docker/docker-compose.yaml:1]: Few things need to be address in future PRs:
1. build-container compatibility
2. test-containers compatibility (tests.yaml)
3. Integration with mkdocs (labels)

File: enterprise/redhat/openshift-ai/gaudi/docker/requirements.txt
Comment (tylertitsworth) [inline enterprise/redhat/openshift-ai/gaudi/docker/requirements.txt:1]: can you add to dependabot.yml?

**Review by tylertitsworth — CHANGES_REQUESTED** [review]
Do we need a plan for OpenShift manifest validation?

File: enterprise/redhat/openshift-ai/gaudi/docker/requirements.txt
Comment (sharvil10) [reply enterprise/redhat/openshift-ai/gaudi/docker/requirements.txt:1]: Added

File: enterprise/redhat/openshift-ai/gaudi/docker/docker-compose.yaml
Comment (sharvil10) [reply enterprise/redhat/openshift-ai/gaudi/docker/docker-compose.yaml:1]: Will do in a future PR.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -95,3 +95,11 @@ updates:
     package-ecosystem: pip
     schedule:
       interval: weekly
+  - directory: enterprise/redhat/openshift-ai/gaudi/docker
+    groups:
+      preset:
```
Comment (tylertitsworth) [inline .github/dependabot.yml:100]: Choose a group name that is relevant.

**Review by tylertitsworth — CHANGES_REQUESTED** [review]
small nit

**Review by tylertitsworth — APPROVED** [review]

**Pull request merged**